osx/setup.sh

```diff
@@ -317,7 +320,7 @@ if [ -e "$DMG_PATH" ]; then
     # codesign the built DMG file
     # unlock the keychain first so we can access the private key
     security unlock-keychain -p $KEYCHAIN_PASSWORD
-    codesign -s "Foundation for Learning Equality, Inc." --force "$DMG_PATH"
+    codesign -s "$SIGNER_IDENTITY_INSTALLER" --force "$DMG_PATH"
```

Comment:
This line is not strictly needed, only when we want the dmg to be signed as well. However, I think this is unnecessary and wouldn't show any warnings (at least not on my side).